rosa_policy/rosa-service-definition.adoc

```diff
@@ -13,3 +13,8 @@ include::modules/rosa-sdpolicy-networking.adoc[leveloffset=+1]
 include::modules/rosa-sdpolicy-storage.adoc[leveloffset=+1]
 include::modules/rosa-sdpolicy-platform.adoc[leveloffset=+1]
 include::modules/rosa-sdpolicy-security.adoc[leveloffset=+1]
+
+
+== Additional resources
```

Comment:
This should be .Additional rather than == Additional 

I think that's why it's not publishing in the correct part of the page.

Reply:
My bad. This info is all in the same module, I thought it was in different modules. So the cross-reference is ok as to where it shows on the page. Just check the formatting for .Additional versus == Additional (it may be a PV1 versus PV2 thing). It publishes fine though.

/lgtm